packages/v4/patternfly-docs/content/accessibility/banner/banner.md

```diff
@@ -3,11 +3,42 @@ id: Banner
 section: components
 ---
 
-**Keyboard users** should not be able to focus on a banner, as it is not an interactive element.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-**Screen reader users** should be able to have a screen reader describe the contents of a banner, but not focus on or interact with a banner otherwise.
+## Accessibility
+
+To implement an accessible PatternFly **banner**:
 
-## Accessibility application:
 - Check the color contrast of a banner and any text on it to make sure they can be read by users with color blindness or other visual impairments.
+- When a banner is used to convey status, add an icon that also conveys status visually, and provide screen reader text using `.pf-screen-reader` or an equivalent.
+
+## Testing
+
+At a minimum, an banner should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-1" label="A user should not be able to focus on the banner with keyboard controls." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-2" label="A user should be able to have a screen reader describe the contents of the banner, but not focus on or interact with the banner otherwise." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="banner-a11y-checkbox-3" label={<span>If the banner conveys status, a status icon and <code>.pf-screen-reader</code> element is included.</span>} />
+  </ListItem>
+</List>
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason  |
+| -- | -- | -- |
+|`screenReaderText`| `Banner` | Text announced by screen readers to indicate the type of banner. Defaults to `${variant} banner` if this prop is not passed in. |
```

Comment:
This has been updated in one of the recent React PRs that got merged so that it not longer has a default. The current prop description is "Text announced by screen readers to indicate the type of banner. This prop should only be passed in when the banner conveys status/severity." Maybe:

```suggestion
Adds text that is accessible only to assistive technologies and is not visually rendered. This prop should only be passed in when the banner is meant to convey status.
```

Or something along those lines.